.github/scripts/build_definition.py

```diff
@@ -15,49 +15,48 @@
-    | macos   | native_mixed       | BP     | BP       |           |             |               | macos-x86_64         |
-    | macos   | iOS_arm64          | dB     | B        |           |             |               |                      |
-    | macos   | iOS_x86_64         | dB     | B        |           |             |               |                      |
-    | macos   | iOS_Mac_ABI        | B      | B        |           |             |               |                      |
-    | macos   | macOS_arm64_static |        |          |           |             |               |                      |
-    | macos   | macOS_arm64_mixed  | BP     | BP       |           |             |               | macos-arm64          |
-    | macos   | macOS_x86_64       | B      | B        |           |             |               |                      |
-    ----------------------------------------------------------------------------------------------
-    |         | flatpak            |        |          |           |             | BP            |                      |
-    |         | native_static      | d      | d        | dBPSD     | dBPSD       |               | linux-x86_64         |
-    |         | native_mixed       | BPS    | BPS      |           |             |               | linux-x86_64         |
-    |         | native_dyn         | d      | d        | dB        | dB          | dBPS          |                      |
+    | bionic  | native_mixed        | BP     |          |           |             |               | linux-x86_64-bionic  |
+    | bionic  | aarch64_mixed       | BP     |          |           |             |               | linux-aarch64-bionic |
+    ----------------------------------------------------------------------------------------------------------------------
+# macOS builds, build binaries on native_dyn and native_static. On another things, build only the libraries
+    | macos   | native_dyn          | d      | d        | dB        | B           |               |                      |
+    | macos   | native_static       |        |          | BP        | BP          |               | macos-x86_64         |
+    | macos   | native_mixed        | BP     | BP       |           |             |               | macos-x86_64         |
+    | macos   | iOS_arm64           | B      | B        |           |             |               |                      |
+    | macos   | iOS_simulator_x86   | B      | B        |           |             |               |                      |
+    | macos   | iOS_simulator_arm64 | B      | B        |           |             |               |                      |
+    | macos   | macOS_arm64         | B      | B        |           |             |               | macos-arm64          |
+    | macos   | macOS_x86           | B      | B        |           |             |               |                      |
```

Comment:
TBH I don't think I fully understand what the differences are between `B`, `dB` and `BP` (yes I did read the keys)

Obviously I am coming from a kiwix for macOS and iOS perspective, so correct me if something should be changed